src/app.js

```diff
@@ -6,23 +6,23 @@ export function app() {
     const response = await fetch('https://pokeapi.co/api/v2/pokemon?limit=5');
     const results = await response.json();
     const pokemons = await results.results;
+
     const pokemonData = [];
 
-    pokemons.forEach(async pokemon => {
-      const response = await fetchPokemon(pokemon);
-      pokemonData.push(response);
-      const pokemonDiv = document.createElement('div');
-      pokemonDiv.innerText = pokemonData.name;
+    pokemons.forEach(pokemon => {
+      // Sorry for the dirty implementation of pictures.
```

Comment:
Don't worry :), it's good that you find out how to extract the id.
See https://github.com/neuefische/cgn/issues/13#issuecomment-596123545 for a similar issue.